ext/storage_sources/s3_store/s3_storage_source.cpp

```diff
@@ -288,13 +308,14 @@ S3Open(WT_FILE_SYSTEM *fileSystem, WT_SESSION *session, const char *name,
             std::cerr << "ss_open_object: GetObject request to s3 failed" << std::endl;
             return (ret);
         }
+        s3->statistics->getObjectCount++;
```

Comment:
Similarly to above; should this still be incremented even if GetObject fails? Or do the statistics just track the number of successful calls?